records_mover/url/resolver.py

```diff
@@ -1,14 +1,125 @@
+from urllib.parse import urlparse
```

Comment:
This module is now responsible for doing the lazy evaluation of the cloud creds.